nc-news-project/src/App.jsx

```diff
@@ -1,12 +1,17 @@
 import { useState } from 'react'
 import Homepage from './components/homepage'
 import './App.css'
+import { Routes, Route } from 'react-router-dom'
+import ArticlePage from './components/ArticlePage'
 
 function App() {
 
   return (
     <>
-     <Homepage/>
+    <Routes>
+        <Route path="/" element={<Homepage />} />
+        <Route path="/:article_id" element={<ArticlePage/>}/>
```

Comment:
might be better to put this on a path of /articles/:article_id as the path you currently have in theory should go to the single article page whatever the user puts in as their route such as /bananas or /2192919